src/cascadia/TerminalApp/ColorScheme.h

```diff
@@ -15,9 +15,10 @@ Author(s):
 
 --*/
 #pragma once
-#include "TerminalSettings.h"
 #include "../../inc/conattrs.hpp"
+#include "inc/cppwinrt_utils.h"
 
+#include <winrt/Microsoft.Terminal.TerminalControl.h>

```

Comment:
does ColorScheme still need this? we removed the reference to ControlSettings from this file